### PR TITLE
Unsubmitted data clean-up

### PR DESCRIPTION
**Pull request by feelfreelinux**
Fixes: issue #1115

### CODE REVIEW

File: app/views/unsubmitted_courses/index.html.haml [hunk 1]
```diff
@@ -10,14 +10,10 @@
       .sort-select
         %select.sorts{:rel => "courses"}
           %option{:rel => "asc", :value => "title"}= t("courses.title")
-          %option{:rel => "desc", :value => "revisions"}= t("metrics.revisions")
-          %option{:rel => "desc", :value => "characters"}= t("metrics.word_count")
-          %option{:rel => "desc", :value => "average-words"}= t("metrics.word_count_average")
-          %option{:rel => "desc", :value => "views"}= t("metrics.view")
-          %option{:rel => "desc", :value => "students"}= t("users.editors")
+          %option{:rel => "desc", :value => "instructor"}= t("courses.instructor.one")
+          %option{:rel => "desc", :value => "tags"}= t("metrics.tags")
```
Comment (mdew192837) [inline app/views/unsubmitted_courses/index.html.haml]: Why does tags go before expected students here and vice versa above? Should it be consistent?

File: config/locales/en.yml [hunk 2]
```diff
@@ -297,6 +297,7 @@ en:
       user_exists: "Sorry, %{username} is not an existing user."
       invalid_language_or_project: Invalid language/project
     expected_students: Expected Students
+    expected_students_doc: Number of expected students count
```
Comment (mdew192837) [inline config/locales/en.yml]: This is redundant. Remove the count at the end or change to something like 'Expected students count'

File: config/locales/en.yml [hunk 3]
```diff
@@ -334,7 +335,7 @@ en:
       your timeline, submit it for approval by Wiki Ed staff. Once approved, you
       will be given an enrollment URL that students can use to join the course.
       (You'll still be able to make edits later.)
-    revisions_doc: The number of edits in the past %{timeframe} days
+    instructor_doc: The name of first instructor
```
Comment (mdew192837) [inline config/locales/en.yml]: probably should be a 'the' in front of 'first'

File: config/locales/en.yml [hunk 4]
```diff
@@ -458,8 +459,10 @@ en:
     char: Chars Added
     date_time: Date/Time
     edit_count_description: Total Edits
+    expected_students: Expected students
```
Comment (mdew192837) [inline config/locales/en.yml]: This message is already on line 299. Is there a reason to add it again?

**Comment by feelfreelinux** [comment]
image

File: config/locales/en.yml [hunk 5]
```diff
@@ -334,7 +335,7 @@ en:
       your timeline, submit it for approval by Wiki Ed staff. Once approved, you
       will be given an enrollment URL that students can use to join the course.
       (You'll still be able to make edits later.)
-    revisions_doc: The number of edits in the past %{timeframe} days
```
Comment (ragesoss) [inline config/locales/en.yml]: This is used elsewhere. Do not delete it.

**Comment by feelfreelinux** [comment]
Looks like tests need fix. Unrelated.

**Comment by ragesoss** [comment]
Good start!

A few problems:

* Sorting by expected students, instructor, and tag do not work. The sorting code for these should just be removed; it's not important to be able to sort by them.
* The mouseover docs for instructors and expected students are not helpful. They should be removed.

**Pull request merged**